samples/Tests/DateTimeTestTranslation.json

```diff
@@ -84,7 +84,7 @@
                 },                                
                 {
                     "type": "TextBlock",
-                    "text": "Missing seconds {{DATE(2017-10-27T22:27Z)}} {{TIME(2017-10-27T22:27:00Z)}}",
+                    "text": "Missing seconds {{DATE(2017-10-27T22:27Z)}} {{TIME(2017-10-27T22:27Z)}}",
```

Comment:
This is actually supposed to be missing the seconds. That payload has 2 groups of formats, the first group (VALID) should all work as intended. The second group (INVALID) are supposed to fail parsing and should return the literal string without any date-formatting logic

Reply:
Yeah. This is in the INVALID group. `{{DATE(…)}}` is missing the seconds but `{{TIME(…)}}` isn't. So, the expected behavior is `output: Missing seconds {{DATE(…)}} 10:27 PM` when it should be `output: Missing seconds {{DATE(…)}} {{TIME(…)}}`.

Removing the seconds from the second part helps test how the `TIME` function handles the same scenario.

Reply:
You'll need to update the UWP TestApp expected results since this will change the rendering of the card as well as the serialization.

---
In reply to: [216468846](https://github.com/Microsoft/AdaptiveCards/pull/1874#discussion_r216468846) [](ancestors = 216468846)